src/Microsoft.ML.Dnn/DnnRetrainTransform.cs

```diff
@@ -903,7 +907,7 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
 
                             var editor = VBufferEditor.Create(ref dst, (int)tensorSize);
 
-                            DnnUtils.FetchData<T>(tensor.ToArray<T>(), editor.Values);
```

Comment:
Is the method `DnnUtils.FetchData` used anymore with this change? I believe it can now be removed.

Reply:
No, it's not used. I will remove it.